color/src/colorspace.rs

```diff
@@ -1716,8 +1716,18 @@ mod tests {
     fn roundtrip() {
         fn test_roundtrips<Source: ColorSpace, Dest: ColorSpace>(colors: &[[f32; 3]]) {
             /// A tight bound on relative numerical precision.
+            #[cfg(not(miri))]
             const RELATIVE_EPSILON: f32 = f32::EPSILON * 16.;
 
+            /// Miri uses precision randomization of floating point operations that do not have a
+            /// guaranteed precision.
+            ///
+            /// Use a lower precision bound on Miri specifically. This is a bit messy, but on our
```

Comment:
I think that "lower" is a potentially confusing term here (given that the absolute value is higher, as it were). Looser is the term I'd normally expect to see here, but it's likely that there's some subtlety here I'm not familiar with.